contracts/message/apps/examples/MsgTest.sol

```diff
@@ -144,6 +140,7 @@ contract MsgTest is MessageApp {
         address // executor
     ) external payable override onlyMessageBus returns (ExecutionStatus) {
         (uint64 n, bytes memory message) = abi.decode((_message), (uint64, bytes));
+        require(keccak256(message) != keccak256(bytes("test101")), MsgDataTypes.REVERT_MSG);
```

Comment:
this may break the sgn e2e tests after contracts bytecode/abi is updated

Reply:
OK. Changed it back to previous version